docs/maps/connect-to-ems.asciidoc

```diff
@@ -0,0 +1,28 @@
+[role="xpack"]
+[[maps-connect-to-ems]]
+== Connecting to Elastic Maps Service
+
+https://www.elastic.co/elastic-maps-service[Elastic Maps Service (EMS)] is a service that hosts
+tile layers and vector shapes of administrative boundaries.
+With Kibana's out-of-the-box settings, the **Maps** application uses EMS services.
+
+Requests to EMS are made with two methods:
+
+* Requests for tile layer meta data, vector shapes meta data, and vector shapes are proxied through the Kibana server.
+* Requests for EMS tiles are made directly from the browser to EMS.
+
+NOTE: The legacy tilemap and regionmap visualizations do not proxy any EMS requests through the Kibana server.
+Requests are made directly from the browser to EMS.
```

Comment:
Maybe we can shorten the Note to one sentence:

The legacy tilemap and regionmap visualizations make requests directly from the browser to EMS.